common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2205,7 +2206,7 @@ public static enum ConfVars {
         "padding tolerance config (hive.exec.orc.block.padding.tolerance)."),
     HIVE_ORC_CODEC_POOL("hive.use.orc.codec.pool", false,
         "Whether to use codec pool in ORC. Disable if there are bugs with codec reuse."),
-    HIVE_ICEBERG_STATS_SOURCE("hive.iceberg.stats.source", "iceberg",
+    HIVE_ICEBERG_STATS_SOURCE(StatsSetupConst.HIVE_ICEBERG_STATS_SOURCE, "iceberg",
```

Comment:
nit: Maybe we can change the `StatsSetupConst.HIVE_ICEBERG_STATS_SOURCE` to `hive.iceberg.stats.source`, though it can pass the compilation, but making the hive-common independent from hive-standalone-metastore-common is important.
Could we move this property to MetastoreConf?

Reply:
this config is used in HiveIcebergStorageHandler as well, so i don't think we could move it. also could we please not ref constants from metastore-common